demo/judgments.py

```diff
@@ -50,27 +53,33 @@ def _judgmentsFromBody(lines):
             yield int(m.group(1)), int(m.group(2)), m.group(3)
 
 
-def judgmentsFromFile(filename):
+def judgments_from_file(filename):
     with open(filename) as f:
-        qidToKeywords = _queriesFromHeader(f)
+        qid_to_keywords = _queries_from_header(f)
     with open(filename) as f:
-        for grade, qid, docId in _judgmentsFromBody(f):
-            yield Judgment(grade=grade, qid=qid, keywords=qidToKeywords[qid], docId=docId)
+        for grade, qid, docId in _judgments_from_body(f):
+            yield Judgment(grade=grade, qid=qid, keywords=qid_to_keywords[qid], doc_id=docId)
 
-def judgmentsByQid(judgments):
-    rVal = {}
-    for judgment in judgments:
+
+def judgments_by_qid(judgments):
+    r_val = {}
+    for j in judgments:
         try:
-            rVal[judgment.qid].append(judgment)
+            r_val[j.qid].append(j)
         except KeyError:
-            rVal[judgment.qid] = [judgment]
-    return rVal
-
+            r_val[j.qid] = [j]
+    return r_val
 
 
 if __name__ == "__main__":
     from sys import argv
-    for judgment in judgmentsFromFile(argv[1]):
-        print(judgment)
 
+    print(len(argv))
 
+    if len(argv) > 1 and len(argv[1]) > 0:
+        judgement_file_name = argv[1]
+    else:
+        judgement_file_name = JUDGMENTS_FILE
+
+    for judgment in judgments_from_file(judgement_file_name):
+        print(judgment)
```

Comment:
In the future, I wonder if we should switch some of these prints to a logging library output?

Reply:
Yes, that is a good one, will have a look. Should not be too hard.